src/main/java/org/symphonyoss/integration/webhook/salesforce/parser/v2/OpportunityNotificationMetadataParser.java

```diff
@@ -75,4 +80,138 @@ protected void postProcessOutputData(EntityObject output, JsonNode input) {
     // Do nothing
   }
 
+  private void proccessNodesObjects(JsonNode node) {
+    if (node.path(SalesforceConstants.OPPORTUNITY_OWNER).getNodeType() == JsonNodeType.MISSING) {
+      ((ObjectNode) node).putObject(SalesforceConstants.OPPORTUNITY_OWNER);
+    }
+
+    if (node.path(SalesforceConstants.OPPORTUNITY_ACCOUNT).getNodeType() == JsonNodeType.MISSING) {
```

Comment:
You could use method JsonNode.isMissingNode() instead of checking through a constant.